lib/ds/RequestExecutor.js

```diff
@@ -90,8 +90,9 @@ RequestExecutor.prototype.execute = function executeRequest(req, callback) {
 
   request(options, function onRequestResult(err, response, body) {
     if (err) {
-      var wrapper = new Error('Unable to execute http request ' + req + ': ' + err.message);
+      var wrapper = new Error('Unable to execute http request ' + req.method + ' ' + req.uri + ': ' + err.message);
```

Comment:
Okay. Then I'll just fix the `req.method`, so that if it is undefined, then it will default to `GET`.